### DISCUSSION TITLE
Wildcard Record Question

### DISCUSSION THREAD

**Discussion by JNDVasco** [BeryJu/gravity#511] [General]
Hello,

I'm trying out gravity before changing all mt DNS servers and everything works great so far except the wildcard records.

It seems to be resolving only  `*.foo.bar` instead of `test1.foo.bar` ... `testN.foo.bar` when it should resolve for all of the options.

Bellow are some screen shots of nslookup and the config I use.

nslookup:
![imagem](https://github.com/BeryJu/gravity/assets/57190680/b7c6bdd0-1564-49ea-a8a5-99d5d9a93d91)

Config: 

![imagem](https://github.com/BeryJu/gravity/assets/57190680/1dee088e-4d44-4254-b36e-94b4489d4a04)

I didn't reopen the issue about wildcard records since nobody has complained, including the creator, so I assume it's a problem on my end. 

What am I missing in my config?

Thanks!

**Comment by BeryJu** [comment]
When adding and testing wildcard queries, I only tested with the etcd handler and not the memory handler, so there was indeed a bug in there: https://github.com/BeryJu/gravity/commit/02e995de70368fed9477b2a52bc24236a39e54fd

I'll publish 0.6.9 after this passes the CI pipeline

**Comment by JNDVasco** [comment]
Sorry for the late response.

Can confirm it's working perfectly now.

Thank you!
